src/main/java/com/lafaspot/pop/client/PopClient.java

```diff
@@ -58,7 +59,6 @@
      */
     public PopClient(final int threads, @Nonnull final LogManager logManager) throws PopException {
```

Comment:
Use Properties class pass in auto-read as a config, pass it in the PopClient c'tor (2nd c'tor)

Reply:
Passing the config in the constructor as Properties.